noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -1002,19 +1002,21 @@ unconstrained fn return_opcode(returndata: [Field]) {}
 unconstrained fn revert_opcode(revertdata: [Field]) {}
 
 #[oracle(avmOpcodeCall)]
-unconstrained fn call_opcode(
+unconstrained fn call_opcode<let N: u32>(
     l2_gas_allocation: u32,
     da_gas_allocation: u32,
     address: AztecAddress,
-    args: [Field],
+    length: u32,
+    args: [Field; N],
 ) {}
```

Comment:
Without context, `length` looks like a redundant parameter given args has `N` size. I guess there's a good reason for this so maybe it could be helpful to add a comment explaining

Reply:
Good point. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/18417/commits/e3eb704b7a9d33fb4cb32508f33281935a8a5ee1 and in https://github.com/AztecProtocol/aztec-packages/pull/18417/commits/e73ead2c19cde8640dd9e58cc7b4a9ffd5ef06ae